model/jpa/src/main/java/org/keycloak/authorization/jpa/entities/PermissionTicketEntity.java

```diff
@@ -35,7 +35,7 @@
  */
 @Entity
 @Table(name = "RESOURCE_SERVER_PERM_TICKET", uniqueConstraints = {
-        @UniqueConstraint(columnNames = {"OWNER", "RESOURCE_SERVER_ID", "RESOURCE_ID", "SCOPE_ID"})
+        @UniqueConstraint(columnNames = {"OWNER", "REQUESTER", "RESOURCE_SERVER_ID", "RESOURCE_ID", "SCOPE_ID"})
```

Comment:
This is not related to map store. I believe this is a bug here and the `REQUESTER` is missing. I created the constraint for map storage based on this line and some test failed because the database contains also `REQUESTER` in uniqueConstraint. See: https://github.com/keycloak/keycloak/blob/master/model/jpa/src/main/resources/META-INF/jpa-changelog-authz-4.0.0.CR1.xml#L52